README.md

```diff
@@ -1,8 +1,34 @@
+## Razer DeathAdder Essential
+
+This fork and branch aim to implement support for Razer DeathAdder Essential.
+
+| Device                                        | USB VID:PID |
+| --------------------------------------------- | ----------- |
+| Razer DeathAdder Essential                    |  1532:006E  |
+
+
+### What works:
+- Setting DPI
+- Setting logo brightness
+- Setting logo effect (none|breathing|static)
+- Setting scroll wheel brightness
+- Setting scroll wheel (none|breathing|static)
+
+### Needs testing:
+- Setting Poll rate
+
+### Known issues:
+- RazorGenie offers the color selection for logo scroll wheel.
+Setting this affects the color but the mouse only changes the amount of green.
+This feature would presumably if the mouse had all LEDs (not just a green one).
+
+- Setting DPI for Y axle has no effect. Not sure if the mouse supports seperate DPIs for X and Y axles.
+
 # [OpenRazer](https://openrazer.github.io/)
 
 A collection of Linux drivers for the Razer devices, providing kernel drivers, DBus services and python bindings to interact with the DBus interface.
 
-**Before raising an issue** saying something doesn't work, read [the troubleshooting wiki page](https://github.com/openrazer/openrazer/wiki/Troubleshooting), try not to create new issues if one exists.
+**Before raising an issue** saying sogit push --set-upstream origin DeathAdder_Essentialmething doesn't work, read [the troubleshooting wiki page](https://github.com/openrazer/openrazer/wiki/Troubleshooting), try not to create new issues if one exists.
 
 Also this is the master branch, devices may appear in the tables below but might not be released in package form yet. Check out the stable branch for what _should_ be in the packages.
 
```

Comment:
For the PR, please revert the readme changes and add the device line into the correct section